tests/integration_tests.py

```diff
@@ -0,0 +1,174 @@
+import json
+import requests
+
+base_url = "http://localhost:8080/ga4gh/drs/v1"
+data_objects_path = "data_objects.json"
+headers = {
+    'accept': 'application/json',
```

Comment:
I think it should be `Accept` with a capital `A`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept